eng/pipelines/mgmt.yml

```diff
@@ -3,16 +3,27 @@ trigger:
     include:
     - master
   paths:
-    exclude:
-    - sdk/
+    include:
+    - '/pom.management.xml'
+    - '/eng/'
+    - '/parent/'
+    - '/profiles/'
+    - '*/resource-manager/*'
```

Comment:
This kind of include filter won't work. It only works by matching the front part of the string with the changes in the commit/PR. Globbing syntax isn't supported (sadly).

Reply:
Got it. I will double check. Alternatively I will try to figure out some way to fast abort the CI if change is not related to management SDK.

Reply:
I'm not sure how much more work it will take but I'd like to see the mgmt build merged with the client/data-plane build ci.yml approach so that we can simply filter by service directory instead of trying to manage these path filters like this.

Reply:
Yes, this is another option.
One concern is that pipeline seems to be created manually (one for each yaml added), at present, and the number of management service build is far greater than client/data-plane build.
Another concern is that this mgmt.yaml pipeline was added quite recently hence I am a bit reluctant to move away from it.

Reply:
@mitchdenny has some tools he can share to help with creating and management of the pipelines.  While the mgmt.yml file was recently created and it was a close translation from what we used to have in jenkins it was always a goal to converge the engineering systems between mgmt and data.

Reply:
Got the background. I could do this.

Reply:
@weidongxu-microsoft check out the pipeline generator tool in https://github.com/azure/azure-sdk-tools. We use this to automatically generate all the Java and JS pipelines for both internal and external builds. I think getting the management plane builds running might just involve adding the management plane libraries to pom.service.xml in the appropriate directories - then it should "just work".

There are things we would need to work out around logistics, for example would management plane like to release on the same cadence as the client libraries. In some ways that makes sense because if there is a new feature that management plane has been modified to support, then shouldn't there be corresponding support in the client libraries (if they exist - they won't in many cases).

I've got an idea. Why don't we get this change over onto a branch called ```feature/something``` and then we can play around with some of these ideas? That branch would include the restructure work you've got here.

Reply:
Agreed. At present state mgmt CI cannot reuse eng\pipelines\templates\stages\archetype-sdk-client.yml directly, hence I will make a feature branch to try out the CI.